sip/sip.rl

```diff
@@ -349,8 +349,8 @@ unquoted_string = ( token LWS )+;
 # stored in Msg by way of type interface and we don't support any types that
 # take parameters.
 ctype_param     = SEMI token EQUAL ( token | quoted_string );
-ctype_mime      = ( token "/" token ) >mark %ContentType;
-ctype          := ctype_mime ctype_param* CRLF @goto_header;
+ctype_mime      = ( token "/" token ) >mark ;
```

Comment:
Should it be

```
ctype_mime      = ( token "/" token ) ;
ctype          := ctype_mime ctype_param*  >mark %ContentType CRLF @goto_header;
```

i.e. putting `>mark` before the captured variable?

Reply:
The original one works, and the updated one also works, but need to add () here.
`
ctype_mime      = ( token "/" token );
ctype          := (ctype_mime ctype_param*) >mark %ContentType CRLF @goto_header;
`

Seems >mark will find the starting point, and  %ContentType will leave the action and update ctype. 

I updated to make it easier to read.